Tfb one pager (#783)
foundersandcoders/coursebook: src/course/handbook/tech-for-better.md

#### [Week 6](../../syllabus/apprenticeship/full-stack-app/schedule/) - [Research Synthesis](https://fac-slides.netlify.app/slides/research-analysis/#0)??

<!-- Rename slides to Research Synthesis for clarity? -->

For each product, a group of developers and the PO write notes on what came out of the research from week 5, before theming the notes and writing insight statements to summarise them. At the end of the session the groups think about any further research to conduct.

#### [Week 7](../../syllabus/projects/in-house-design/schedule/) - [Product Management](https://fac-slides.netlify.app/slides/product-management/#0)

#### [Week 8](../../syllabus/projects/in-house-build-1/schedule/) - Selection (Software Developers only)

Software developers review all Tech for Better Product ideas as a group and shortlist by confirming their individual preferences.

#### [Week 9](../../syllabus/projects/in-house-build-2/schedule/) - [Analysis Workshop](https://fac-slides.netlify.app/slides/analysis/#0)

> Following the shorlisting and selection of products in weeks 8, Product Owners work with their assigned Product Team (software developers) to take a closer look at their product idea.

Reviewer: ```suggestion
Following your selection of Products in week 8, chosen Product Owners work with their assigned Product Team to analyse the product idea.
```

Reviewer: A Product Team is PO and developers, a development team is just dev's

Together they plan the main path through their application - considering how users will find their service, how users will learn what the service offers and how to use it, as well as what steps users will take when using the app.

> The PO and Product Team also think about building a product which will respond to the user needs, and consider the intended and unintended consequences of their product.

Reviewer: ```suggestion
The Product Team also think about building a product which will respond to the user needs, and consider the intended and unintended consequences of their product.
```

I'm suggesting we remove this exercise and include user story mapping #780

#### [Week 10](../../syllabus/projects/tfb-design/schedule/) - TfB Design Week

During the first half of TfB Design Week, the Product Team (software developers) work alongside their own Product Owner(PO) to define their MVP. Sessions include:

**[Definition Workshop](https://fac-slides.netlify.app/slides/definition/#0):** Product Teams and PO search for inspiration from other sources before generating their own ideas and creating low fidelity wireframes.

**[Wireframing](https://www.figma.com/blog/how-to-wireframe/#types-of-wireframes):** Product Teams work together to create a high fildelity prototype of their app, refining it following feedback from there PO and end users.

> **[User Testing](https://fac-slides.netlify.app/slides/ut-synthesis/#0):** Product Teams conduct user testing with real users scheduled by their PO, using their high fidelity protoype.

Reviewer: ```suggestion
**[Usability Testing](https://fac-slides.netlify.app/slides/usability-testing/#0):** Product Teams conduct user testing with real users scheduled by their PO, using their high fidelity protoype.
```

During the second part of the week, the Product Teams turn their attention to the technical apsects of their product. Sessions include:

> **[Code Planning](https://fac-slides.netlify.app/slides/code-planning/#0):** This in two parts. During Service Blueprinting the Product Teams identify points of interation between users and their app, and document what should happen in the back end. During Teck Stacks, the Product Teams then identify potential technical options for their project's requirements.

Reviewer: ```suggestion
**[Code Planning](https://fac-slides.netlify.app/slides/code-planning/#0):** Create a Service Blueprint which identifies points of interaction between your users and the app, and how the app will respond to user input. Identify a tech stack to meet the project requirements.
```

> **[Technical Spikes](../../syllabus/projects/TFB-design/spikes/):** Product Teams read and explore documentation on technologies that might be used for their project, or to find a definitive answer for some of the "How might we" questions that came up during the Definition Workshop.

Reviewer: ```suggestion
**[Technical Spikes](../../syllabus/projects/TFB-design/spikes/):** Product Teams read and explore documentation on technologies that might answer some of the "How might we..." questions from the Definition Workshop.
```

Right now, HMW happens in the Analysis workshop. We could move it to the Definition between Inspiration and Sketching

Author: My reference to the Definition Workshop, rather than Analysis, came from the Tech Spikes tab on TfB Design week - http://localhost:8080/course/syllabus/projects/TFB-design/spikes/ . If we do not move it, this will need changing too.

#### [Week 11](../../syllabus/projects/tfb-build-1/schedule/) / [Week 12](../../syllabus/projects/tfb-build-2/schedule/) - TfB Build Week 1 / TfB Build Week 2

> Product Teams build their MVP across two build weeks, in partnership with their Product Owners. Each Product Team is also supported through check-in calls and code reviews with volunteer Project Mentors from apprentice employers and FAC alumni.

Reviewer: ```suggestion
Build your MVP across two build sprints, in partnership with your Product Owners. You'll have support and code reviews from [Project Mentors](/course/handbook/mentor-roles/).
```

Can you double check the link here before merging? I forget if we need a `.` before the first `/`

> At the end of Build Week 2, the Product Teams and PO's present their products to local leaders in the social tech sector and alumni from Founders and Coders at the Tech for Better Showcase.

Reviewer: ```suggestion
At the end of Build Week 2, you'll present your MVP at the Tech for Better Showcase.
```
